POEApi.Model/Gear.cs

```diff
@@ -67,5 +70,51 @@ public int NumberOfSockets()
                 return components;
             }
         }
+
+        public override string PobData
+        {
+            get
+            {
+                try
+                {
+                    var pobData = new StringBuilder();
+                    pobData.AppendLine(Name);
+                    pobData.AppendLine(TypeLine);
+                    pobData.AppendLine($"Unique ID: {Id}");
+                    pobData.AppendLine($"Item Level: {ItemLevel}");
+                    pobData.AppendLine($"Quality: {Quality}");
+
+                    if (Sockets != null && Sockets.Any())
+                    {
+                        var socketData = string.Join("=", Sockets.Select(x => x.ToPobFormat()));
```

Comment:
Add a comment explaining the format required by POB.  Are there any distinctions between links that are present and those that are missing?

Reply:
Great spot - I hadn't realised that slots which aren't linked are being linked up. *Fixed*